Add etcd to troubleshooting, datastore reference (#2805)
sensu/sensu-docs: content/sensu-go/6.1/operations/deploy-sensu/deployment-architecture.md

## Hardware sizing

> Because etcd's design prioritizes consistency across a cluster, the speed with which write operations can be completed is very important to the performance and health of a Sensu cluster. 

Reviewer: ```suggestion
Because etcd's design prioritizes consistency across a cluster, the speed with which write operations can be completed is very important to the Sensu cluster's performance and health.
```

This means that you should provision Sensu backend infrastructure to provide sustained IO operations per second (IOPS) appropriate for the rate of observability events the system will be required to process.

To maximize Sensu Go performance, we recommend that you:

>  * Follow our [recommended backend hardware requirements][1] documentation.

Reviewer: ```suggestion
 * Follow our [recommended backend hardware configuration][1].
```

 * Implement [documented etcd system tuning practices][11].
 * [Benchmark your etcd storage volume][12] to establish baseline IOPS for your system.

>  * [Scale event storage using Postgres][13], reducing the overall volume of etcd transactions.

Reviewer: ```suggestion
 * [Scale event storage using PostgreSQL][13] to reduce the overall volume of etcd transactions.
```
